java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerRepository.java

```diff
@@ -28,7 +28,7 @@ public String createNewCustomer(@RequestBody CustomerRequestBody customerRequest
     if(rows > 0){
       log.info("A new customer was CREATED!!!(has been inserted)");
     }
-    return sql;
+    return "Welcome "+ customerRequestBody.getFirstName() + "" + customerRequestBody.getLastName();
```

Comment:
hey Daniel. We talked about this last week. RestFul APIs should not be returning a string. Look back at what we said last week, as to what this API should return. Seems like you weren't paying attention...